impl/sql/mutationstorage/queue.go

```diff
@@ -187,8 +187,8 @@ func readQueueMessages(rows *sql.Rows) ([]*mutator.QueueMessage, error) {
 	return results, nil
 }
 
-// DeleteMessages removes messages from the queue.
-func (m *Mutations) DeleteMessages(ctx context.Context, domainID string, mutations []*mutator.QueueMessage) error {
+// DeleteMessages removes messages from the log.
```

Comment:
I look forward to seeing the implementation of this one :)

Reply:
haha.  This reminds me that I need to delete this.

Reply:
#1067